scapy/data.py

```diff
@@ -30,7 +30,7 @@
 SOL_PACKET = 263
 # From asm/socket.h
 SO_ATTACH_FILTER = 26
-SO_TIMESTAMPNS = 35
+SO_TIMESTAMPNS = 35  # SO_TIMESTAMPNS_OLD: not 2038 safe
```

Comment:
What do you mean?

Reply:
See https://github.com/torvalds/linux/blob/a2d79c7174aeb43b13020dd53d85a7aefdd9f3e5/include/uapi/asm-generic/socket.h#L109-L115

- `SO_TIMESTAMPNS_OLD = 35` returns timespecs that use long to store the date = not 2038 safe
- `SO_TIMESTAMPNS_NEW` returns timespecs that are long long
- `SO_TIMESTAMPNS` defaults for the old one

We would use the "NEW" one but it's very recent and isn't supported well yet.
https://lwn.net/Articles/776219/